modules/manage/partials/rbac-dp.adoc

```diff
@@ -60,6 +172,31 @@ rpk security acl create --operation write --topic private-software-versions --al
 
 endif::[]
 
+== RBAC best practices
+
+Follow these recommendations for effective role-based access control:
+
+Role design
+
+* Use descriptive names: Choose role names that clearly indicate their purpose (`DataEngineers`, `ReadOnlyAnalysts`)
+* Follow job functions: Align roles with actual job responsibilities and organizational structure
+* Keep roles focused: Create specific roles rather than overly broad ones (`TopicReaders` vs `SuperUsers`)
+* Plan for growth: Design roles that can accommodate new team members and evolving needs
+
+Permission management
+
+* Start with minimal permissions: Grant only the access required for the role's function
+* Use wildcards carefully: Prefixed patterns like `analytics-*` are useful but review regularly
+* Avoid DENY rules: Prefer specific ALLOW rules over complex DENY/ALLOW combinations
```

Comment:
Should the rules always be in inline code formatting?

Reply:
I'll update!